engine/src/main/java/com/datatorrent/stram/StramClient.java

```diff
@@ -454,6 +456,7 @@ public void startApplication() throws YarnException, IOException
         appPath = new Path(configuredAppPath);
       }
       String libJarsCsv = copyFromLocal(fs, appPath, localJarFiles.toArray(new String[]{}));
+      processSSLConfig(dag.getValue(Context.DAGContext.SSL_CONFIG), fs, appPath, localResources);
```

Comment:
I am with @vrozov on having a separate method - looks cleaner and easy to maintain. I guess the compiler can inline the method as an optim if needed.